README.md

```diff
@@ -31,13 +31,13 @@ keytool -importkeystore -deststorepass your_password -destkeypass your_password
 rm keystore.p12 key.pem cert.pem
 ```
 
-You also need to grant the `java` executable permission to bind to ports below 1024 without running as root. To do this, run the following as root or via sudo:
+Optional: You also need to grant the `java` executable permission to bind to ports below 1024 without running as root. To do this, run the following as root or via sudo:
```

Comment:
I would rephase to "To use port 443 (the browser default) for https, you will need to ..."